lib/3scale/backend/alert_limit.rb

```diff
@@ -9,7 +9,7 @@ class AlertLimit
       attr_accessor :service_id, :value
 
       def save
-        storage.sadd(key_allowed_set(service_id), value.to_i) if valid?
+        storage.sadd?(key_allowed_set(service_id), value.to_i) if valid?
```

Comment:
@jlledom How did you figure out which calls needed to be updated (from `sadd` to `sadd?`) and which are not? Failing tests or smth else?

Cause there are still a lot of occurrences of `sadd` (for example). I guess it's fine as long as we're not expecting a boolean from them. But how can we make sure?

Reply:
> How did you figure out which calls needed to be updated (from sadd to sadd?) and which are not? 

By reading the [Changelog](https://github.com/redis/redis-rb/blob/master/CHANGELOG.md), from **4.1.3** to **5.2.0**.

For `:pipelined` and `:exists?` it's easier because I replaced all occurrences. In the case of `:sadd` and `:sadd?` (same for `:srem[?]`). What I did is check all calls, and add the `?` to those where the return value is being used and needs to be a boolean. Then passing tests and making a lot of manual testing in different scenarios. There are not many occurrences of those, so it's doable.